tests/pr_test.go

```diff
@@ -10,76 +10,62 @@ import (
 )
 
 // Use existing resource group
-const basicExampleDir = "examples/basic"
-const completeExampleDir = "examples/complete"
+const rootDaDir = "./"
+
+// Use existing resource group for test
+const resourceGroup = "default"
 
 // Current supported regions
 var validRegions = []string{
 	"us-south",
 	"eu-de",
 }
 
-func setupOptionsBasicExample(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
+func setupOptionsRootDA(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
 	options := testhelper.TestOptionsDefault(&testhelper.TestOptions{
 		Testing:      t,
 		TerraformDir: dir,
 		IgnoreDestroys: testhelper.Exemptions{ // Ignore for consistency check
 			List: []string{
-				"module.watsonx_saas.module.configure_user.null_resource.configure_user",
-				"module.watsonx_saas.module.configure_user.null_resource.restrict_access",
+				"module.configure_user.null_resource.configure_user",
+				"module.configure_user.null_resource.restrict_access",
 			},
 		},
 		IgnoreUpdates: testhelper.Exemptions{ // Ignore for consistency check
 			List: []string{
-				"module.watsonx_saas.module.configure_user.null_resource.configure_user",
-				"module.watsonx_saas.module.configure_user.null_resource.restrict_access",
+				"module.configure_user.null_resource.configure_user",
+				"module.configure_user.null_resource.restrict_access",
 			},
 		},
 	})
+
 	terraformVars := map[string]interface{}{
-		"location": validRegions[rand.Intn(len(validRegions))],
+		"location":            validRegions[rand.Intn(len(validRegions))],
+		"resource_group_name": resourceGroup,
```

Comment:
The DA is creating the resource group by default. So its going to have to be a unique name. Suggest to use `options.Prefix` here